R/extract.data.R

```diff
@@ -22,6 +22,8 @@ NULL
     # name, type, typeSignature
     c('character', 'character', 'list_named')
   )
+  # Make sure column names are unique
+  column.info[['name']] <- make.names(column.info[['name']], unique=TRUE)
```

Comment:
This breaks the column naming for the returned values. 

The issue stems from the `dplyr::bind_rows` bug/"feature" of deduplicating columns with the same name. The fix therefore should be in `.fetch.all`. It probably would rather need to be a workaround where you do not use `bind_rows` if there are duplicate column names.